### PR TITLE
Refactor common labels

### PR DESCRIPTION
**Pull request by omegas27**

### CODE REVIEW

File: service/controller/resource/servicemonitor/service/apiserver.go [hunk 1]
```diff
@@ -37,14 +37,6 @@ func APIServer(cluster metav1.Object, provider string, installation string) *pro
...
 							TargetLabel:  "app",
 						},
-						{
-							SourceLabels: []string{"__meta_kubernetes_namespace"},
-							TargetLabel:  "namespace",
-						},
-						{
-							SourceLabels: []string{"__meta_kubernetes_pod_name"},
-							TargetLabel:  "pod_name",
-						},
```
Comment (TheoBrigitte) [inline service/controller/resource/servicemonitor/service/apiserver.go:47]: Why are those gone ?

File: files/templates/_common.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{{- define "_node_role" -}}
```
Comment (TheoBrigitte) [inline files/templates/_common.yaml:1]: Please move this to a `_node_role.yaml` to keep things oraganized.

**Review by TheoBrigitte — APPROVED** [review]
Good to see more label alignement

File: service/controller/resource/servicemonitor/service/apiserver.go (hunk 1 above)
Comment (omegas27) [reply service/controller/resource/servicemonitor/service/apiserver.go:47]: Prometheus operator is adding them for ServiceMonitors

Comment (omegas27) [reply service/controller/resource/servicemonitor/service/apiserver.go:47]: https://github.com/prometheus-operator/prometheus-operator/blob/55baf034c431ed2c78d950b187f7d8b34dd06860/pkg/prometheus/promcfg.go#L567

**Pull request merged**